app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -259,8 +269,51 @@ Kong Error
 No API key found in request.
 ```
 
+## Migrate configuration between runtime groups
+
+You can also use decK to migrate or duplicate configuration between runtime
+groups.
+
+1. Export configuration from the original runtime group with
+[`deck dump`](/deck/latest/reference/deck_dump) into a state file:
+
+    ```bash
+    deck dump \
+      --konnect-runtime-group default \
+      --output-file default.yaml
+    ```
+
+1. In the file, change the runtime group name to the new group:
```

Comment:
@haydenlam Why do we need to change `runtime_group_name` in the file if we're providing `--konnect-runtime-group` to deck?

Reply:
yeah this shouldn't be necessary, let me know otherwise!

Reply:
It is currently, using your latest test branch. If I _don't_ change the runtime group name in the state file, diff works fine. Sync also looks like it should work, but I only get a diff response from it:

```
Summary:
  Created: 0
  Updated: 8
  Deleted: 0
```

When I check the target runtime group with a `deck dump` or through the UI, I see that no changes have been applied.

However, if I set the name explicitly in the state file:
```
_konnect:
  runtime_group_name: test
```
And then `deck sync` that, I get the response I'm looking for:
```
creating consumer consumer
creating service test3
creating service test2
creating service test4
creating service MyService
creating service example_service4
creating service MyService_1
creating route test5
Summary:
  Created: 8
  Updated: 0
  Deleted: 0
```

Reply:
Interesting, I'll look into this!

Reply:
Looking at it again, I think that in the first situation, decK is diffing the runtime group set in the file? And completely ignoring the `--konnect-runtime-group` flag. So, if the group name in the file and the group name specified by flag are mismatched, nothing happens.

Reply:
One more thing: if I completely remove the `runtime_group_name` from the state file and only set it in the flag, then things work fine.

Reply:
> If the group name in the file and the group name specified by flag are mismatched, nothing happens.

Yep, this is correct, the user's intention is unclear therefore we take no action. (Ideally error out)

> If I completely remove the _konnect section from the state file and only set the RG in the flag, then things work fine.

This is also the expected behavior.

Reply:
> Yep, this is correct, the user's intention is unclear therefore we take no action. (Ideally error out)

If that's the case, then for this doc, it makes sense to include changing the name of the RG in the file in these instructions. If the purpose is to migrate between RGs, you will have to either change or remove it either way.

Reply:
Yep, agree, for the reason that the `_konnect. runtime_group_name ` is present in the file from the `deck dump`.

Reply:
> Yep, this is correct, the user's intention is unclear therefore we take no action. (Ideally error out)
> 
> >

@lena-larionova @haydenlam  Can you please pull the latest from my branch? I just implemented this.

```
_format_version: "1.1"
_konnect:
  runtime_group_name: test
...
...
```

```
$ ./deck sync --konnect-addr https://us.api.konghq.tech --konnect-runtime-group prod
Error: Warning: Runtime Group 'prod' specified via --konnect-runtime-group flag is different from 'test' found in state file(s).
```